auto-voice-channels.py

```diff
@@ -643,10 +643,19 @@ class MyClient(discord.AutoShardedClient):
 
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
+        self.last_ready = None
+
+    def up_time(self):

```

Comment:
@gregzaal Lines 646 -> 653, No idea why it always does this even tho that was the only thing that got change, unless pycharm reformatting weirdly?